cache-filter/src/configuration.rs

```diff
@@ -7,13 +7,16 @@ pub struct FilterConfig {
     pub failure_mode_deny: bool,
     /// Number of retries for setting data to cache
     pub max_tries: u32,
+    /// Max memory in bytes that shared data is allowed to use.
+    pub max_shared_memory_bytes: u64,
 }
 
 impl Default for FilterConfig {
     fn default() -> Self {
         FilterConfig {
             failure_mode_deny: true,
             max_tries: 5,
+            max_shared_memory_bytes: 4294967296, // equivalent to 4GB
```

Comment:
Never hardcode these, always use a constant defined in a prominent place.

Reply:
constant added